src/__tests__/lib/exampleData.js

```diff
@@ -45,7 +45,7 @@ const makeUniqueRandInt = (itemsType: string, end: number): (() => number) => {
 export const randString = () => randInt(2 ** 54).toString(36);
 
 const randUserId: () => number = makeUniqueRandInt('user IDs', 10000);
-const userOrBotProperties = ({ name: _name }) => {
+const userOrBotProperties = ({ name: _name, email: _email }) => {
```

Comment:
No need for the initial `_` on `_email`. (We only have one on `name` because of the lint error that would otherwise result.)